tst/dogfood.tst

```diff
@@ -0,0 +1,62 @@
+#############################################################################
+##
+##  AutoDoc package
+##
+##  Test the behavior of AutoDoc by generating its own manual
```

Comment:
While in principle I think this is a good idea, the one drawback is that any change to the manual will lead to a test failure, unless one remember to update the dogfood test, too.

But the "fix" for that shouldn't be too hard in each instance, so I am not saying this is a major issue, just pointing it out.